src/components/OrdersWidget/index.tsx

```diff
@@ -245,6 +274,36 @@ const OrdersWidget: React.FC = () => {
       {!noOrders && networkId && (
         <OrdersForm>
           <form action="submit" onSubmit={onSubmit}>
+            <FilterTools
+              customStyles={
+                isWidget &&
+                `
+                #filterLabel {
+                  bottom: -1.2rem;
+                }
+
+                .balances-searchTokens { 
+                    height: 3.6rem; 
+                    margin: 0.8rem; 
+                    width: 100%; 
+                    
+                    > input { 
+                      width: 100%; 
+                    } 
+                }`
+              }
```

Comment:
Wouldn't be better to have a different widget instead of inlining the style, even if as a prop?

Reply:
oki